### PR TITLE
Add en ios course

### PR DESCRIPTION
**Pull request by bhumiiagarwal**
## What does this PR do?
Add resource

### Description
Added a course in english on iOS

### Why is this valuable (or not)?
Yes, it is valuable.

### How do we know it's really free?
It is available on coursera

### For book lists, is it a book? For course lists, is it a course? etc.
it is a course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by bhumiiagarwal** [comment]
@eshellman @sohansai please review this PR

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -854,6 +854,7 @@
 * [SwiftUI Tutorials for Beginners](https://www.youtube.com/playlist?list=PLMRqhzcHGw1Z-lZaaun3A3mV9PbEfHANI) - CodeWithChris
 * [The Complete Swift iOS Developer - Create Real Apps in Swift](https://www.udemy.com/course/the-complete-ios-10-developer-course/) - Grant Klimaytys (Udemy)
 * [Unit Testing in iOS](https://www.youtube.com/playlist?list=PLMRqhzcHGw1ZLLvLwuW8AP3n-A3nsRn9P) - CodeWithChris
+* [Get Started with iOS Development (iOS 13, Swift 5)](https://www.youtube.com/playlist?list=PLSzsOkUDsvdu5Mm67aBYs2YPu2OM4mFzt) - London App Brewery
```
Comment (sohansai) [inline courses/free-courses-en.md:857]: Follow alphabetical ordeing

Comment (bhumiiagarwal) [reply courses/free-courses-en.md]: updated

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**